staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -49,6 +49,16 @@ type AggregationManager interface {
 	UpdateAPIServiceSpec(apiServiceName string, spec *spec.Swagger, etag string) error
 	RemoveAPIServiceSpec(apiServiceName string) error
 	GetAPIServiceInfo(apiServiceName string) (handler http.Handler, etag string, exists bool)
+
+	// Recorded delegation API services' name. If delegation API services' OpenAPI
+	// spec is dynamic (e.g. apiExtensionsServer), the recorded delegationAPIServices
+	// can be exposed to aggregation controller, which periodically try syncing the
+	// OpenAPI specs based on the API services' name as key
+	GetDelegationAPIServicesName() []string
```

Comment:
why a slice if the func gives a name?

Reply:
Sorry it meant to give a list of names. Currently it holds the apiservice names of two delegation apiservers (generic apiserver and apiextensions apiserver). I updated the func name and documentation.